asset_delivery.md

```diff
@@ -1,6 +1,10 @@
 # Asset Delivery
 
-## Option A - Use Cloudfront CDN and Cloudflare CDN
+## Option A - Use Cloudfront Directly
+
+Since Cloudfront offers simple SSL certs, this now is an option.
```

Comment:
no, there are no restrictions (yet?). imo it's not needed, but let's discuss this face to face :) (actually, this doesn't concern this particular pull request - it's already the case with option b)

the difference from option c is that cloudflare is more difficult to configure and understand, and does not allow some configurations, e.g. timeouts (at least for me)